src/azure-cli-core/azure/cli/core/tests/test_command_registration.py

```diff
@@ -170,7 +170,6 @@ def _mock_load_command_loader(loader, args, name, prefix):
         class TestCommandsLoader(AzCommandsLoader):
 
             def load_command_table(self, args):
-                super(TestCommandsLoader, self).load_command_table(args)
```

Comment:
@jsntcy Nice catch!

Reply:
Looks like there are still other places using super.load_command_table(),could you search and remove them as well?

---
In reply to: [459828026](https://github.com/Azure/azure-cli/pull/13879#discussion_r459828026) [](ancestors = 459828026)

Reply:
This PR is only for removing the extra debug log entries. We can handle them when time permits.